streams/no.m3u

```diff
@@ -13,20 +13,40 @@ http://media1.adventist.no:1935/live/hope2/playlist.m3u8
-#EXTINF:-1 tvg-id="NRKSuper.no",NRK Super [Geo-blocked]
-https://nrk-nrksuper.akamaized.net/23/0/hls/nrk_super/playlist.m3u8
-#EXTINF:-1 tvg-id="TV2Nyhetsstrommen.no",TV2 Nyhetsstrømmen (720p)
-https://ws15-hls-live.akamaized.net/out/u/1153546.m3u8
-#EXTINF:-1 tvg-id="TV2Sport1.no",TV 2 Sport 1 (720p)
-https://ws31-hls-live.akamaized.net/out/u/1416253.m3u8
+#EXTINF:-1 tvg-id="NRK1.no",NRK 1 (1080i) [Geo-blocked]
+https://nrk-live-no.akamaized.net/nrk1_dk1/muxed.m3u8
+#EXTINF:-1 tvg-id="NRK1Nordnytt.no",NRK 1 (Nordnytt) (1080i) [Geo-blocked]
+https://nrk-live-no.akamaized.net/nrk1_dk2/muxed.m3u8
+#EXTINF:-1 tvg-id="NRK1Nordlan.no",NRK 1 (Nordland) (1080i) [Geo-blocked]
+https://nrk-live-no.akamaized.net/nrk1_dk3/muxed.m3u8
+#EXTINF:-1 tvg-id="NRK1Midtnytt.no",NRK 1 (Midtnytt) (1080i) [Geo-blocked]
+https://nrk-live-no.akamaized.net/nrk1_dk4/muxed.m3u8
+#EXTINF:-1 tvg-id="NRK1MoreOgRomsdals.no",NRK 1 (Møre og Romsdal) (1080i) [Geo-blocked]
+https://nrk-live-no.akamaized.net/nrk1_dk5/muxed.m3u8
+#EXTINF:-1 tvg-id="NRK1Vestlandsrevyen.no",NRK 1 (Vestlandsrevyen) (1080i) [Geo-blocked]
+https://nrk-live-no.akamaized.net/nrk1_dk6/muxed.m3u8
+#EXTINF:-1 tvg-id="NRK1Rogaland.no",NRK 1 (Rogaland) (1080i) [Geo-blocked]
+https://nrk-live-no.akamaized.net/nrk1_dk7/muxed.m3u8
+#EXTINF:-1 tvg-id="NRK1Agder.no",NRK 1 (Agder) (1080i) [Geo-blocked]
+https://nrk-live-no.akamaized.net/nrk1_dk8/muxed.m3u8
+#EXTINF:-1 tvg-id="NRK1VestfoldOgTelemark.no",NRK 1 (Vestfold og Telemark) (1080i) [Geo-blocked]
+https://nrk-live-no.akamaized.net/nrk1_dk9/muxed.m3u8
+#EXTINF:-1 tvg-id="NRK1Innlandet.no",NRK 1 (Innlandet) (1080i) [Geo-blocked]
+https://nrk-live-no.akamaized.net/nrk1_dk10/muxed.m3u8
+#EXTINF:-1 tvg-id="NRK2.no",NRK 2 (1080i) [Geo-blocked]
+https://nrk-live-no.akamaized.net/nrk2/muxed.m3u8
+#EXTINF:-1 tvg-id="NRK3.no",NRK 3 (1080i) [Geo-blocked]
+https://nrk-live-no.akamaized.net/nrk3/muxed.m3u8
+#EXTINF:-1 tvg-id="NRKSuper.no",NRK Super (1080i) [Geo-blocked]
+https://nrk-live-no.akamaized.net/nrksuper/muxed.m3u8
+#EXTINF:-1 tvg-id="NRKTegnspraak.no",NRK Tegnspråk (1080i) [Geo-blocked]
+https://nrk-live-no.akamaized.net/nrk_tegnspraak/muxed.m3u8
 #EXTINF:-1 tvg-id="TVModum.no",TV Modum (720p)
 https://cdn010.panaccess.com/5677_streams/TVMODUM/index.m3u8
 #EXTINF:-1 tvg-id="TVOstfold.no",TV Østfold (1080p)
 https://streamnorge.com:8088/hls/tvo.m3u8
-#EXTINF:-1 tvg-id="SunnmoreLIVE.no@SD" http-referrer="https%3A%2F%2Fslive.no%2F",Sunnmore LIVE
+#EXTINF:-1 tvg-id="SunnmoreLIVE.no@SD" http-referrer="https%3A%2F%2Fslive.no%2F",Sunnmøre LIVE
 #EXTVLCOPT:http-referrer=https%3A%2F%2Fslive.no%2F
 https://live03.delivery.twentythree.com/live/amlst:ids06jcunt260s3olais9ck283:68173018-4011529d07c63c7759cd.mp4/playlist.m3u8?referer=https%3A%2F%2Fslive.no%2F&uuid=cbe9a779-6917-09f8-8f17-7a011ce0b34c
+#EXTINF:-1 tvg-id="TV2Nyhetsstrommen.no",TV2 Nyhetsstrømmen (720p)
+https://ws15-hls-live.akamaized.net/out/u/1153546.m3u8

```

Comment:
403. TV2 Play seems to have a login system. Try and see if there's anything "adaptive" before.

Reply:
They added a non-login version at https://www.tv2.no/video/live/nyhetsstrommen/20259505 sometime in the past year or 2, but I was not able for the life of me to get a consistently working .m3u8 from it.